amazon-personalize-ranking/src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/reranker/impl/AmazonPersonalizedRankerImpl.java

```diff
@@ -96,7 +100,14 @@ public SearchHits rerank(SearchHits hits, PersonalizeRequestParameters requestPa
 
             SearchHits personalizedHits = combineScores(hits, result);
             return personalizedHits;
-        } catch (Exception ex) {
+        } catch (AmazonServiceException e) {
+            logger.error("Exception while calling personalize campaign: {}", e.getMessage());
```

Comment:
nit: Let's call it ``Personalize service`` instead of ``campaign``.

Reply:
Updated it in the revision